### PR TITLE
Kotlin/Java SDK version bumps, fix accidentally hidden line

### PR DESCRIPTION
**Pull request by nathan-contino-mongo**
### Jira

- no JIRA, just an issue I noticed based on the feedback widget

### Staged Changes (Requires MongoDB Corp SSO)

- [Android SDK - Encryption](https://docs-mongodbcom-staging.corp.mongodb.com/realm/docsworker-xlarge/padding-android-encrypt/sdk/android/advanced-guides/encryption/)

### Review Guidelines

[REVIEWING.md](https://github.com/mongodb/docs-realm/blob/master/REVIEWING.md)

### CODE REVIEW

File: examples/android/sync/app/src/androidTest/java/com/mongodb/realm/examples/kotlin/EncryptTest.kt [hunk 1]
```diff
@@ -171,7 +171,8 @@ class EncryptTest : RealmTest() {
                 "realm_key",
                 KeyProperties.PURPOSE_ENCRYPT or KeyProperties.PURPOSE_DECRYPT)
                 .setBlockModes(KeyProperties.BLOCK_MODE_CBC)
-                .setEncryptionPaddings(KeyProperties.ENCRYPTION_PADDING_PKCS7) // :hide-start:
+                .setEncryptionPaddings(KeyProperties.ENCRYPTION_PADDING_PKCS7)
```
Comment (MongoCaleb) [inline examples/android/sync/app/src/androidTest/java/com/mongodb/realm/examples/kotlin/EncryptTest.kt:174]: Is any change needed to the docs to explain this code change?

**Comment by MongoCaleb** [comment]
Staging link is borked, but I don't think that changes the review.

**Review by MongoCaleb — APPROVED** [review]
LGTM with one question.

File: examples/android/sync/app/src/androidTest/java/com/mongodb/realm/examples/kotlin/EncryptTest.kt (hunk 1 above)
Comment (nathan-contino-mongo) [reply examples/android/sync/app/src/androidTest/java/com/mongodb/realm/examples/kotlin/EncryptTest.kt:174]: It's already in the Java version of this snippet -- I just accidentally let the "hide-start" bluehawk annotation trim it from the Kotlin version when I converted between languages. Silly mistake on my part that just broke the Kotlin version. So no!

**Pull request merged**